spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/external-config.adoc

```diff
@@ -35,7 +35,7 @@ Config data files are considered in the following order:
 . <<features#features.external-config.files.profile-specific,Profile-specific application properties>> outside of your packaged jar (`application-\{profile}.properties` and YAML variants).
 
 NOTE: It is recommended to stick with one format for your entire application.
-If you have configuration files with both `.properties` and `.yml` format in the same location, `.properties` takes precedence.
+If you have configuration files with both `.properties` and `.yaml` format in the same location, `.properties` takes precedence.
```

Comment:
Both `.yml` and `.yaml` are still supported. I think it would be better to say "and YAML format".